lib/spacesuit/auth_middleware.ex

```diff
@@ -31,7 +21,10 @@ defmodule Spacesuit.AuthMiddleware do
     end
   end
 
-  defp handle_bearer_token(req, env, token) do
+  defp handle_bearer_token(session_service_enabled, request, environment, token)
```

Comment:
So, it seemed like the decision of whether or not to deal with the token could be moved to the `handle_bearer_token` function. I split the 2 not-success cases into separate function heads and then created a logic-less head at the top to make it clear, using variable naming, what should be passed in.